liveRecorder/scripts/build_ts2mp4_convertor.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+if [ "$#" -lt 2 ]; then
+	echo "usage build_ts2mp4_convertor <ffmpeg build path> <product path>"
+	exit
+fi
+
+FFMPEG_BUILD_PATH=$1
+PRODUCT_ROOT_PATH=$2
+TARGET=ts_to_mp4_convertor
+CONVERTOR_DIR=${PRODUCT_ROOT_PATH}/${TARGET}
+
+[ `uname` = "Linux" ] && OS="linux" || OS="darwin"
+
+export FFMPEG_BUILD_PATH
+export OS
+
+echo "OS=$OS"
+
+
+if [ -e $CONVERTOR_DIR ]; then
+	pushd $CONVERTOR_DIR
+		mkdir -p obj
+		[ -e  ${TARGET} ] && rm -f ${TARGET}
+		[ -e  obj/${TARGET} ] && rm -f obj/${TARGET}
+
+		echo "starting to build ${TARGET}"
+
+		make
+
+		if [ -s  obj/${TARGET} ] ; then
+			echo "**************************************************************************************"
+			echo "${TARGET} was built successfully, copying to bin folder"
+			echo "**************************************************************************************"
+			mkdir -p bin
+			echo "cp obj/${TARGET} bin"
+			cp obj/${TARGET}  ../bin
```

Comment:
You shouldn't be manually calling cp from the shell script, the Makefile should have a "install" target.